webpack.common.js

```diff
@@ -16,32 +16,27 @@ module.exports = {
   },
   module: {
     rules: [
-      {
-        test: require.resolve("./src/theme.js"),
-        use: "imports-loader?this=>window"
-      },
```

Comment:
This didn't seem to be needed anymore, seem to work fine without? could remove `imports-loader`